src/plugins/kibana_usage_collection/README.md

```diff
@@ -1,14 +1,18 @@
 # Kibana Usage Collection
```

Comment:
I thought it was easier to follow in a table, with a small summary of what each collector does. I also added some missing ones like Usage Counters and the new one this PR implements.